hazelcast/src/main/java/com/hazelcast/map/impl/query/QueryOperation.java

```diff
@@ -123,28 +78,8 @@ public ExceptionAction onInvocationException(Throwable throwable) {
         return super.onInvocationException(throwable);
     }
 
-    @Override
```

Comment:
method not needed anymore. We'll send a response regardless instead of secretly trying to send a response if the method 'returns no response'